src/transactions/feed/DepositOrWithdrawFeedItem.tsx

```diff
@@ -0,0 +1,172 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import AppAnalytics from 'src/analytics/AppAnalytics'
+import { HomeEvents } from 'src/analytics/Events'
+import TokenDisplay from 'src/components/TokenDisplay'
+import Touchable from 'src/components/Touchable'
+import { navigate } from 'src/navigator/NavigationService'
+import { Screens } from 'src/navigator/Screens'
+import { Colors } from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import variables from 'src/styles/variables'
+import TransactionFeedItemImage from 'src/transactions/feed/TransactionFeedItemImage'
+import { DepositOrWithdraw, TokenTransactionTypeV2 } from 'src/transactions/types'
+
+interface DescriptionProps {
+  transaction: DepositOrWithdraw
+}
+
+function Description({ transaction }: DescriptionProps) {
+  const { t } = useTranslation()
+  const appName = transaction.appName
+  let title
+  let subtitle
+
+  switch (transaction.type) {
+    case TokenTransactionTypeV2.Deposit:
+      title = t('transactionFeed.depositTitle')
+      subtitle = t('transactionFeed.depositSubtitle', { appName })
+      break
+    case TokenTransactionTypeV2.Withdraw:
+      title = t('transactionFeed.withdrawTitle')
+      subtitle = t('transactionFeed.withdrawSubtitle', { appName })
+      break
+  }
+
+  return (
+    <View style={styles.contentContainer}>
+      <Text style={styles.title} testID={'DepositOrWithdrawFeedItem/title'} numberOfLines={1}>
+        {title}
+      </Text>
+      {!!appName && (
+        <Text
+          style={styles.subtitle}
+          testID={'DepositOrWithdrawFeedItem/subtitle'}
+          numberOfLines={1}
+        >
+          {subtitle}
+        </Text>
+      )}
+    </View>
+  )
+}
+
+interface AmountDisplayProps {
+  transaction: DepositOrWithdraw
+  isLocal: boolean
+}
+
+function AmountDisplay({ transaction, isLocal }: AmountDisplayProps) {
+  let amountValue
+  let tokenId
+
+  switch (transaction.type) {
+    case TokenTransactionTypeV2.Deposit:
+      amountValue = new BigNumber(-transaction.outAmount.value)
+      tokenId = transaction.outAmount.tokenId
+      break
+    case TokenTransactionTypeV2.Withdraw:
+      amountValue = new BigNumber(transaction.inAmount.value)
+      tokenId = transaction.inAmount.tokenId
+      break
+  }
+
+  const textStyle = isLocal
+    ? styles.amountSubtitle
+    : [
+        styles.amountTitle,
+        transaction.type !== TokenTransactionTypeV2.Deposit && { color: Colors.primary },
```

Comment:
```suggestion
        transaction.type !== TokenTransactionTypeV2.Deposit && { color: Colors.accent },
```
I assume this has to be green, which is now named accent, and maybe we can check for `= withdraw` instead of `!= deposit`?

Reply:
Ah yes, but the Colors name change hasn't been merged yet.